2_dataCleaning/DataCleaning_Code.Rmd

```diff
@@ -272,6 +272,22 @@ mydata <- mydata %>% mutate(Mean = rowMeans(cbind(Day1, Day2, Day3, Day4)))
 
 ```
 
+### Combining data from multiple sources
+
+Sometimes, data might be spread across multiple spreadsheets, and you'll want to combine those for your analysis. For example, maybe this study had a follow-up survey on Day 30. Scores from that survey were entered into another spreadsheet, which only has the subject ID and that score. We want to include that score into our data. Here I'm using the function `merge()`, which is one of the base functions in R, not part of the tidyverse.
```

Comment:
`merge()` seems fine here. participants can also be told that there are joining functions, e.g. `left_join()`, in the tidyverse if they like?